java/ql/lib/change-notes/2021-11-04-log-injection-query.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
```

Comment:
Change note should probably be in `java/ql/src/change-notes` rather than `java/ql/lib/change-notes`.

Reply:
🤦‍♂️ you're right, thanks. Fixed.